o8g/Extractor/setguids.xml

```diff
@@ -0,0 +1,12 @@
+﻿<xml>
+  <sets>
+      <set id="gen-con-2017" number="00" value="b9353360-2f08-4517-a7d0-960b085d6ba4" />
```

Comment:
Nevermind, just missed the core set. Wondering if this gencon set is needed though? None of them are new cards, they're just duplicates. Do you have both for thrones so that people can choose if they want to use the promotional cards rather than having to override a single card to switch between them?

Reply:
Yeah its supposed to be a promo set but it doesnt seem like theres any images for it.  I added it for completion’s sake, not against removing it

Reply:
Yah after trying the image fetcher they're still proxies. Maybe leave it till they add some? Seems confusing for folks atm